src/encoding.md

```diff
@@ -543,6 +544,60 @@ Ashley-Zurich-92";
 # quick_main!(run);
 ```
 
+[ex-csv-filter]: #ex-csv-filter
+<a name="ex-csv-filter"></a>
+## Filter CSV records matching a predicate
+
+[![csv-badge]][csv] [![cat-encoding-badge]][cat-encoding]
+
+_Disclaimer: this example is modified version of [this](https://docs.rs/csv/*/csv/tutorial/index.html#filter-by-search) one._
```

Comment:
I would move the disclaimer to bottom of the text or example and reword the text to:
"_Disclaimer: this example has been adapted from the [csv crate tutorial](https://docs.rs/csv/*/csv/tutorial/index.html#filter-by-search) ._"